package.nls.pl.json

```diff
@@ -16,5 +16,7 @@
     "jupyter.command.jupyter.interruptkernel.title": "Przerwij Jupyter Kernel",
     "jupyter.command.jupyter.restartkernel.title": "Restartuj Jupyter Kernel",
     "jupyter.command.jupyter.expandallcells.title": "Rozwiń wszystkie komórki IPythona",
-    "jupyter.command.jupyter.collapseallcells.title": "Zwiń wszystkie komórki IPythona"
+    "jupyter.command.jupyter.collapseallcells.title": "Zwiń wszystkie komórki IPythona",
+    "jupyter.command.jupyter.runallcellsabove.title": "Uruchom komórkę powyżej",
+    "jupyter.command.jupyter.runcellandallbelow.title": "Uruchom komórkę poniżej"
```

Comment:
I noticed that all of the commands in this language only have the first letter uppercased, except for the name `Jupyter`, so I decided to not uppercase the cases I'm fixing in English.

(I did add these two localizations. I thought it would only be good to do so)

Should I make the other words uppercase in Dutch and Polish?

Reply:
I wouldn't touch these unless you know what it says.

Reply:
ok undoing these